src/openads/infrastructure/logger/LogLevelLoggerFactory.js

```diff
@@ -1,7 +1,9 @@
 import LogLevelLogger from './LogLevelLogger'
+import LoggerFactory from '../../domain/logger/LoggerFactory'
 
-export default class LogLevelLoggerFactory {
+export default class LogLevelLoggerFactory extends LoggerFactory {
```

Comment:
Nailed! 
![](https://media.giphy.com/media/7vhAnGwSOQvUQ/giphy.gif)

Reply:
I know it has been only to put a free meme ;)

![](https://media.giphy.com/media/lbqVdsFBjCCjK/giphy.gif)